onchain/src/nfts/canvas_nft.cairo

```diff
@@ -97,4 +114,21 @@ mod CanvasNFT {
             self.nfts.emit(CanvasNFTMinted { token_id, metadata });
         }
     }
+
+    #[abi(embed_v0)]
+    impl CanvasNFTLikeAndUnlike of ICanvasNFTLikeAndUnlike<ContractState> {
+        fn like_nft(ref self: ContractState, token_id: u256) {
+            self
```

Comment:
Good work. Could you add asserts, so users can only like and unlike nfts within the store. ( check against nft count )

Reply:
you mean check that nft exist in the store using the token_id

Reply:
Yeah, token_id should be less than nft count

Reply:
i see,